hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -130,6 +130,12 @@ SqlResult execute(CreateIndexPlan plan) {
             // common case. There's no atomic operation to create an index in IMDG, so it's not easy to
             // implement.
             MapContainer mapContainer = getMapContainer(hazelcastInstance.getMap(plan.mapName()));
+
+            // Very rare case, but still may happens...
+            if (mapContainer == null) {
+                throw QueryException.error("Can't create index: mapping '" + plan.mapName() + "' not initialized yet");
```

Comment:
We must create the index also in this case. The fix might be using `MapServiceContext.getMapContainer(mapName)` instead of `MapServiceContext.getMapContainers().get(mapName)`, as we're doing currently.